apps/src/code-studio/pd/pre_workshop_survey/PreWorkshopSurvey.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import FormController from '../form_components/FormController';
+import PreWorkshopQuestions from './PreWorkshopQuestions';
+
+export default class PreWorkshopSurvey extends FormController {
+  /**
+   * @override
+   */
+  getPageComponents() {
+    return [
+      PreWorkshopQuestions
+    ];
+  }
+
+  isLocalSummer() {
+    return this.props.course === "CS Principles" && this.props.subject === "5-day Summer";
+  }
```

Comment:
is this needed?

Reply:
Nope! leftover from copy/pasting another form.